Geometry/EcalMapping/src/EcalElectronicsMapping.cc

```diff
@@ -1095,7 +1098,8 @@ void EcalElectronicsMapping::GetListofFEDs(const EcalEtaPhiRegion& region, std::
 		}
 		int minR = minL + 18;
 		int maxR = maxL + 18;
-		int iR = minR;
+		// FIXME: Value stored to 'iR' during its initialization is never read
+		int iR; // = minR;
```

Comment:
Since iR is only used in the while loop, you can just declare it there. Since the value is an int and will be on the stack anyway there would be no performance gain from having it outside the loop.